src/_bentoml_sdk/images.py

```diff
@@ -275,11 +275,6 @@ def get_image_from_build_config(
             "python.wheels is not supported by bento v2, fallback to bento v1"
         )
         return None
-    if python_options.pack_git_packages:
```

Comment:
Why do we remove this?

Reply:
the default value is true, this will cause all fallback to v1